File: ApiProject/ApiProject/Models/Appointment.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ApiProject.Models
{
    public class Appointment
    {
        [Key]
        public Guid Id { get; set; }
        public DateTime Date { get; set; }
        public TimeSpan StartTime { get; set; }
        public TimeSpan EndTime { get; set; }
        public bool Available { get; set; }
        public string HerId { get; set; }
        public string BookedBy { get; set; }
        public string Exercuted { get; set; }

        public Guid UpgradeId { get; set; }
        [ForeignKey("UpgradeId")]
        public Upgrade UpgradeFk { get; set; }
    }
}

File: BookingUpgrade-master/src/app/models/Appointment.ts
import { Time } from "@angular/common";

export interface Appointment
{   
    //id: guid
    versionNumber: string;
    date : string;
    hour: string; //Time?
    durationMin: string; //Number
    available:boolean;
    herId?:string;
    bookedBy?: string;
    executed: boolean;
}